6.Knative_with_Cloud_Resource_App/Components/component-knative-configuration-flask-app.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: core.oam.dev/v1alpha2
+kind: Component
+metadata:
+  name: webapp
+spec:
+  workload:
+    apiVersion: serving.knative.dev/v1
+    kind: Configuration
+    metadata:
+      name: webapp
+      namespace: default
+    spec:
+      template:
+        spec:
+          containers:
+            - image: zzxwill/postgresql-flask-web-application:v0.1
```

Comment:
better to use oamdev account

Reply:
Sure, would you please help grant me with write authority?